cpp/src/tests/CMakeLists.txt

```diff
@@ -125,19 +125,19 @@ set(KATZ_TEST_SRC
 ###################################################################################################
 # - gdfgraph tests --------------------------------------------------------------------------------
 
-set(GDFGRAPH_TEST_SRC
-    "${CMAKE_CURRENT_SOURCE_DIR}/gdf_graph/gdf_graph.cu")
+#set(GDFGRAPH_TEST_SRC
+#    "${CMAKE_CURRENT_SOURCE_DIR}/gdf_graph/gdf_graph.cu")
 
```

Comment:
There are many stand-alone tests in there that don't need GRMAT

Reply:
I'm not sure: what seems to be the issue here? I commented out GRMAT tests and `grmat.cu`, as requested by @BradReesWork.

Reply:
By commenting tests apps here you also got rid of all non-grmat tests. If you check https://github.com/rapidsai/cugraph/blob/branch-0.10/cpp/src/tests/gdf_graph/gdf_graph.cu you will see that it contains only 3 tests using `GRMAT` and 22 tests that don't.

Reply:
Ah, okay, I understand now. Okay, will address this.